ec_grasp_planner/src/planner.py

```diff
@@ -189,14 +166,29 @@ def handle_run_grasp_planner(self, req):
             print("No object was detected")
             return plan_srv.RunGraspPlannerResponse("", -1)
 
-        robot_base_frame = self.args.robot_base_frame
-        object_frame = objects[0].transform
+        robot_base_frame = self.args.robot_base_frame       
 
         time = rospy.Time(0)
         graph.header.stamp = time
-        object_frame.header.stamp = time # TODO why do we need to change the time in the header?
-        bounding_box = objects[0].boundingbox
+        
+        self.tf_listener.waitForTransform(robot_base_frame, "/ifco", time, rospy.Duration(2.0))
+        ifco_in_base = self.tf_listener.asMatrix(robot_base_frame, Header(0, time, "ifco"))
```

Comment:
Is there any reason why you inverted the transform?
On master we use 
`lookupTransform('ifco', robot_base_frame,  rospy.Time(0))`
which represents the transformation from robot base frame to ifco.

You are using
`asMatrix(robot_base_frame, Header(0, time, "ifco"))`
which represents the transformation from ifco to robot base

The transform is used in multi_object_params.py and that change would require a second somehow unintuitive inversion.

Reply:
So, to clarify things... To our understanding, we are computing the tf of the ifco frame with respect to the base coordinate frame, while you were using the inverse of that. 

We think that computing this tf is more intuitive because the base coordinate frame is fixed while the ifco frame is moving. Sorry we missed the inversion that you are mentioning. For us the phrase x_in_y meant "frame x in the y coordinate frame" so we thought we were computing the same frame.